docs/controls/access_control/ac-02.md

```diff
@@ -0,0 +1,60 @@
+# Access Control
+## AC-02 - Account Management
+The organization:
+
+a. Identifies and selects the following types of information system accounts to support organizational missions/business functions: (individual, shared, group, system, guest/anonymous, emergency, developer/manufacturer/vendor, temporary, and service);
+
+b. Assigns account managers for information system accounts;
+
+c. Establishes conditions for group and role membership;
+
+d. Specifies authorized users of the information system, group and role membership, and access authorizations (i.e., privileges) and other attributes (as required) for each account;
+
+e. Requires approvals by (applicable information system managers) for requests to create information system accounts;
+
+f. Creates, enables, modifies, disables, and removes information system accounts in accordance with (the processes defined in the OS Standard Operating Procedures for Information Security: Access Control document (see Section AC-2));
+
+g. Monitors the use of information system accounts;
+
+h. Notifies account managers:
+
+   1. When accounts are no longer required;
+   2. When users are terminated or transferred; and
+   3. When individual information system usage or need-to-know changes;
+
+i. Authorizes access to the information system based on:
+
+   1. A valid access authorization;
+   2. Intended system usage; and
+   3. Other attributes as required by the organization or associated missions/business functions;
+
+j. Reviews accounts for compliance with account management requirements [at least every 180 days]; and
+
+k. Establishes a process for reissuing shared/group account credentials (if deployed) when individuals are removed from the group.
+
+For CSP Only: AC-2 (j) [at least annually]
+
+## TDP Implementation
+a. The TDP system accounts are OFA Admin and Data Prepper.  
```

Comment:
I think we should capture the `System Owner`/`Product Owner` role as well - have we landed on a consistent name for that role?

Reply:
Part `a.` also makes mention of developers, so that role should likely be captured here as well.

Reply:
@hursey013 I wasn't sure about this. In a practical sense developers have a role but there is no role in the system defined for developers. Does that make a difference?

Reply:
Yeah I agree they don't have a role at the application level, but from a system level they do. I don't think we need to go into much detail beyond identifying that the developer role exists, and that the management of those accounts will use functionality inherited by GitHub/Cloud.gov.

Reply:
It might make sense to agree on some verbiage that distinguishes a role defined in the system, such as `OFA Admin` or `Data Prepper` as opposed to a role that someone plays, but it not defined in the system, such as `Developer` or `System Owner`. In English, it makes sense to say that there is a role the developer plays, I just want to make sure it's not conflated with the others in our readers' understanding.

Reply:
Updated Part a. to include System Admin, OFA Admin, Data Preppers, and Developers